server/tests/cluster.rs

```diff
@@ -101,3 +103,35 @@ async fn cluster_name_mismatch() -> googletest::Result<()> {
 
     Ok(())
 }
+
+#[test(tokio::test)]
+async fn replicated_loglet() -> googletest::Result<()> {
+    let mut base_config = Configuration::default();
+    base_config.bifrost.default_provider = ProviderKind::Replicated;
```

Comment:
[⬇️ Low prio] I've found reducing the number of partitions to greatly help with log readability but maybe we specifically want to stress the scheduler here. (Requires making `bootstrap_num_partitions` pub.)

```suggestion
    base_config.bifrost.default_provider = ProviderKind::Replicated;
    base_config.common.bootstrap_num_partitions = NonZeroU16::new(1).unwrap();
```

Reply:
No special intention to stress test the scheduler here. I was mainly lazy and tried avoiding to type more 😅 Will change it.